src/tests/vendor_agnostic/fixtures.py

```diff
@@ -0,0 +1,385 @@
+import time
+from contextlib import contextmanager
+import shutil
+import pandas as pd
+import pytest
+from sqlalchemy import MetaData, create_engine
+from sql import _testing
+from sqlalchemy.ext.declarative import declarative_base
+import uuid
+from dockerctx import new_container
+
+
+# Skip the test case when live mode is on (with --live arg)
+@pytest.fixture(scope="session")
+def skip_on_live_mode(pytestconfig):
+    if pytestconfig.getoption("live"):
+        pytest.skip("Skip on live mode")
```

Comment:
Probably we can move fixtures `skip_on_live_mode` and `skip_on_local_mode` into src/tests/conftest.py, so that `src/tests/integration` and `src/tests/vendor_agnostic` can reuse those

Reply:
Another way is to move `src/tests/vendor_agnostic` into `src/tests/integration`, so you don't have to create duplicated fixtures